.github/workflows/example.yml

```diff
@@ -22,14 +22,21 @@ jobs:
           - macOS-latest
           - windows-latest
 
-        resolver:
-          - nightly   # ghc-9.4 (as of writing)
-          - lts       # ghc-9.2 (as of writing)
-          - lts-20.2  # ghc-9.2
-          - lts-19.33 # ghc-9.0
-          - lts-18.28 # ghc-8.10
-          - lts-16.31 # ghc-8.8
-          - lts-12.26 # ghc-8.4
```

Comment:
I know it seems redundant, but can you keep this `resolver` key in addition to `include`? It causes the matrix jobs to come out named as just `test ({resolver})`, instead of the `test ({resolver}, {ghc})`, like you get when you only define the `include`. I'd rather not have to mess with the Required Statuses to match different names.

Reply:
Hmm, with `runner` in the mix the test names are coming out even differently. Not sure what to do :thinking:

Reply:
Wow, nice find!

Reply:
So this did still result in changed test names,

```
Example / test (macOS-latest, lts-20.2, 9.2.5) (pull_request) Successful in 2m 
```

Instead of the expected,

```
test (macOS-latest, lts) Expected — Waiting for status to be reported
```

I was hoping to avoid that. Not only would I rather not change the required statuses, I'd also like to be able to continue to require the `lts` test for each OS rather than having to hard-code `lts-20.2` and maintain it over time.

I'm going to keep thinking on this a bit.

Reply:
Seems like https://github.com/freckle/stack-action/pull/35/commits/c3d40010117b3db59abcc8dfd780b4f1716e33e0